chef/cookbooks/network/attributes/default.rb

```diff
@@ -35,6 +35,7 @@
     default[:network][:ovs_service] = "openvswitch-switch"
   else
     default[:network][:ovs_service] = "openvswitch"
+    default[:network][:ovs_pkgs] = ["openvswitch"]
```

Comment:
I personally would prefer to move the "exception" into the "legacy" code branch. e.g. changing line 22 above to only have "openvswitch" and add an extra if for suse < 12.3 pushing the additional package (or just not worry about that part at all and just change the default globally).